Portal/vip-api/src/main/java/fr/insalyon/creatis/vip/api/business/AuthenticationBusiness.java

```diff
@@ -62,7 +62,6 @@ public void authenticateSession(String userName, String password) throws ApiExce
              User user = configurationBusiness.signin(userName, password);
              
              AbstractAuthenticationService.setVIPSession(apiContext.getRequest(), apiContext.getResponse(), user);
-             configurationBusiness.updateUserLastLogin(userName);
```

Comment:
This is just a duplicate removal, the same method is called in setVIPSession (the line before).